alicloud/resource_alicloud_instance_test.go

```diff
@@ -1063,16 +1063,44 @@ variable "name" {
 	default = "tf-testAccInstanceConfigMultipleRegions"
 }
 
+resource "alicloud_vpc" "vpc_foo" {
+  name = "${var.name}"
+  provider = "alicloud.beijing"
+  cidr_block = "192.168.0.0/16"
+}
+
+resource "alicloud_vpc" "vpc_bar" {
+  name = "${var.name}"
+  provider = "alicloud.shanghai"
+  cidr_block = "192.168.0.0/16"
+}
+
+resource "alicloud_vswitch" "vsw_foo" {
+  provider = "alicloud.beijing"
+  vpc_id = "${alicloud_vpc.vpc_foo.id}"
+  cidr_block = "192.168.0.0/24"
+  availability_zone = "${data.alicloud_zones.default.zones.0.id}"
```

Comment:
Please set resource `alicloud_vswitch` name using `name = "${var.name}"` to ensure it can be sweeped.